src/components/dashboard/FeaturedApps/FeaturedApps.tsx

```diff
@@ -39,12 +34,9 @@ const onWcWidgetClick = (e: SyntheticEvent) => {
 }
 
 export const FeaturedApps = ({ stackedLayout }: { stackedLayout: boolean }): ReactElement | null => {
-  const router = useRouter()
-  const [featuredApps, _, remoteSafeAppsLoading] = useRemoteSafeApps(SafeAppsTag.DASHBOARD_FEATURED)
+  const txBuilder = useTxBuilderApp()
   const enableWc = useHasFeature(FEATURES.NATIVE_WALLETCONNECT)
```

Comment:
nit: Unrelated but this variable name is confusing like its a user action to enable WC. Something like `isWcEnabled` would be more descriptive